news/2017-06-27-beatmap-spotlight-may-2017.md

```diff
@@ -26,96 +27,98 @@ First of all, congratulations to the winners of the April 2017 Spotlights. They
-
-This mapset offers more of what we've come to expect from Ascendance with a well crafted series of difficulties that are a pleasure to play. All difficulties do a nice job of building up to the energetic kiai where the Evanesce difficulty stands out from the rest with its quick doubles that kick the difficulty up a notch. Overall while the mapset may not offer many surprises, its solid execution makes for a welcome addition to the catch spotlights.
+Overall, the map is quite challenging for its star rating, being harder than other mapsets people usually see in the Ranked category. It also features several mappers, providing players with different styles across all difficulties. It's quite the engaging map, and one that could surely make you wiggle from joy!
 
 [![](/wiki/shared/news/2017-06-27-beatmap-spotlights-may-2017/shelter.jpg)](https://osu.ppy.sh/s/536749)
 
-Despite of this set is being Myle's very first ranked set it came out to be a really enjoyable one. The spread built up to grew quite constantly. I really enjoyed playing it, he made a good work.
+Despite this set is being **[Myle](https://osu.ppy.sh/users/4915649)**'s very first ranked set it came out to be a really enjoyable one. The spread built up to grew quite constantly. He learned how to make the rhythm to be consistent and yet still provide variable patterns in a short time. The flow of the difficulties are just splendid and perfectly executed. The Platter is really made for the targeted audience, he held back with the HyperJumps and only placed one where it should be which gives better impact in overall and feels really good. It was really enjoying to play, he made a good work!
 
 ## osu!mania
 
 [![](/wiki/shared/news/2017-06-27-beatmap-spotlights-may-2017/trigger.jpg)](https://osu.ppy.sh/s/555076)
 
-I have to say it. This map is definetly the most fun experience I had in a while from the ranked pool and I can only say thanks and congratulations to Wonki for winning this month spotlights and also for providing us such a good piece. The SVs on this map feels intense but they are not overdone and it just feels natural with the song, making it really confortable to play. Combine that with a cool usage of minijacks at the very end with some simple LN patterns this map just has everything you want to be the map of the month... and its art!
+This beatmap set by **[Wonki](https://osu.ppy.sh/users/7898495)** definitely is a fun experience from the ranked pool and congratulations to Wonki for winning this month spotlights and also for providing us such a good piece. The scroll speed changes on this map feels intense but they are not overdone and it just feels natural with the song, making it really confortable to play. Combine that with a cool usage of minijacks at the very end with some simple long note patterns this map just has everything you want to be the map of the month... and its art!
```

Comment:
Still lots of issues with this, at least at the start.

Congratulations to Wonki for winning this month's spotlights for mania with Trigger, an entertaining and fresh experience rife with some cleverly applied mapping techniques that fit the music perfectly. Scroll speed is used masterfully, lending an air of intensity to the set without it being overdone, and also keeping in line with the song's natural ebb and flow. Smart usage of minijacks at the very end and some simple long note patterning gives this map a touch of everything you'd want to see in a good osu!mania chart, and it's even artistic to boot!